Sources/SafeDICore/Errors/FixableInjectableError.swift

```diff
@@ -41,38 +41,32 @@ public enum FixableInjectableError: DiagnosticError {
     // MARK: - InjectableDiagnosticMessage
 
     private struct InjectableDiagnosticMessage: DiagnosticMessage {
-        var diagnosticID: MessageID {
```

Comment:
these computed properties were never executed in tests. There's no good way to do so as part of an integration test. So now we're just creating them up-front.